layouts/partials/header.html

```diff
@@ -1,57 +1,40 @@
 <!doctype html>
 
-<html lang="{{ .Site.LanguageCode | default "en"}}">
+<html lang="{{ .Site.LanguageCode | default " en "}}">
 
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>{{ .Site.Title }}</title>
-    <meta name="description" content="The HTML5 Herald">
-    <meta name="author" content="{{ .Site.Author.name }}">
-    {{ .Hugo.Generator }}
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab|Ruda" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="{{ "css/styles.css" | relURL}}" />
+  <title>{{ .Site.Title }}</title>
+  {{ partial "meta" . -}}
```

Comment:
is the `-` necessary?

Reply:
I will double-check, I believe I added them to collapse an empty space after the partial.